### ISSUE TITLE
Searching PS3 and PSVita Video Games using Universal Search endpoint.

### ISSUE THREAD

**Issue by isFakeAccount** [isFakeAccount/psnawp#52]
_Originally posted by @glebmiller in https://github.com/isFakeAccount/psnawp/discussions/44_

PlayStation API universal search endpoint does not return any PS3 or PSVITA games. This happens even on the app itself. Due to this the `trophy_titles_for_title` method cannot be used since we don't have title ids. 

> Note: If you get all of the trophy titles using the `client.trophy_titles(limit=100)` method, then it will include the PS3 and PSVITA trophies.

**Comment by platinumachievements** [comment]
Hey, is this still a thing that we can’t access ps3 and Vita trophies? I think this is the only thing holding me back jumping fully into this api.

**Comment by isFakeAccount** [comment]
> Hey, is this still a thing that we can’t access ps3 and Vita trophies? I think this is the only thing holding me back jumping fully into this api.

@platinumachievements What is your gamertag and can you see ps3 and ps vita trophies on PlayStation Android/iOS app? 

If you can see your trophies on Android/iOS app then I can add it to api.

**Comment by platinumachievements** [comment]
Thanks for the quick reply, yeah I can see the ps3 trophies for my games on the IOS app. I dont have any Vita trophies, so they must be able to be gotten, I would need to get other peoples ps3 trophies not just Mine

**Comment by isFakeAccount** [comment]
> @isFakeAccount Thanks for the quick reply, yes I can see the ps3 trophies for my games on the IOS app. I dont have any Vita trophies so I can’t check that. so they must be able to be gotten, I would need to get other peoples ps3 trophies not just Mine. I can also see friends ps3 trophies via the app as well

@platinumachievements If your friends have ps vita then it would help a lot. Also, can you add me as friend. 

Gamertag: VaultTec-Co

**Comment by platinumachievements** [comment]
@isFakeAccount added, I’ll see if I can find anyone with vita/psp. Just scroll to the bottom of my list for ps3 games.

**Comment by platinumachievements** [comment]
@isFakeAccount i can see friends Vita games as well, they are under PSVITA.

**Comment by isFakeAccount** [comment]
@platinumachievements You can use this code example to get all your PS Trophies 

```py
from os import getenv

from dotenv import load_dotenv
from psnawp_api import PSNAWP
from psnawp_api.models.trophies import PlatformType

load_dotenv()


def main() -> None:
    psn = PSNAWP(getenv("NPSSO_CODE", "NPSSO_CODE"))
    client = psn.me()

    print("Trophy Titles")
    trophy_titles = client.trophy_titles(limit=100)
    for trophy_title in trophy_titles:
        if PlatformType.PS3 in trophy_title.title_platform:
            print(trophy_title.title_name, trophy_title.title_platform, trophy_title.earned_trophies, trophy_title.np_communication_id)

    print("\nTrophies")
    for trophy in client.trophies("NPWR05167_00", PlatformType.PS3, limit=10):
        print(trophy.trophy_name)

    print("\nTrophy Group Summary")
    the_lego_movie_tgs = client.trophy_groups_summary("NPWR05167_00", platform=PlatformType.PS3, include_progress=True)
    print(the_lego_movie_tgs)


if __name__ == "__main__":
    main()
```

The main issue is that you cannot search for PlayStation games using Search class. Since you can't find title ids using search function, you can't use `client.trophy_titles_for_title` because it requires title ids. 

But remaining methods and endpoints work fine.

**Comment by platinumachievements** [comment]
Thanks! I think I understand the limitations now, you can retrieve them, but can’t individual search for them as you can’t retrieve the title ID.

**Comment by isFakeAccount** [comment]
> Thanks! I think I understand the limitations now, you can retrieve them, but can’t individual search for them as you can’t retrieve the title ID.

Yeah pretty much. You have to fetch all and manually filter `trophy_titles`. Once you have the `np_communication_id` then you don't need the title_ids anyways. @platinumachievements

**Comment by AhmedGIbrahim** [comment]
Hello can you check that please the PlayStation plus price in turkey region is rising one year ago for 1400 TL, so i know someone buy with the old price 240 TL![image](https://github.com/user-attachments/assets/0e61888a-e27e-498d-bda4-9a787e17e3ea)

**Comment by AhmedGIbrahim** [comment]
IP9102-NPIA90006_01-PSPLUSSUBS365DAY
Product is still working for subscribe for 12 months woth old price but I don't know how i get it
